doc/sphinxman/source/scf.rst

```diff
@@ -579,17 +579,23 @@ that |PSIfour| expects the numpy file on disk to have the ``.npy`` extension, no
 Convergence Stabilization
 ~~~~~~~~~~~~~~~~~~~~~~~~~
 
-With regard to convergence stabilization, Pulay's Direct Inversion of the
-Iterative Subspace (DIIS) extrapolation,  Gill's Maximum Overlap Method (MOM),
-and damping are all implemented. A summary of each is presented below,
+A summary of Psi's supported convergence stabilization techniques is presented below:
 
 DIIS [On by Default]
-    DIIS uses previous iterates of the Fock Matrix together
+    DIIS uses previous iterates of the Fock matrix together
     with an error criterion based on the orbital gradient to produce an informed
     estimate of the next Fock Matrix. DIIS is almost always necessary to converge
     the SCF procedure and is therefore turned on by default. In rare cases, the
     DIIS algorithm may need to be modified or turned off altogether, which may be
     accomplished via :term:`options <DIIS (SCF)>`.
+ADIIS [On by Default]
+    ADIIS uses previous iterates of the Fock and density matrices to produce an
+    informed estimate of the next Fock matrix. ADIIS estimates are based on minimizing
+    the energy rather than zeroing the residual, so this performs best in the early
```

Comment:
minimizing a proxy energy

Reply:
This is very technical, but reworded "minimizing an energy estimate" rather than proposed rewording.

In the HF case, both ADIIS and EDIIS minimize the HF energy functional, which isn't a proxy.

Reply:
Well... ADIIS and EDIIS do ultimately reproduce the HF energy once they have accumulated enough data. IIRC you need to have a history of `N` Fock matrices in a basis set with `N` functions to recover the HF energy. Before that, the energy functionals are not the same!

Reply:
Fine, ADIIS and EDIIS minimize the HF energy functional _on a subset of its domain of definition_. Either way, I think "minimizing an energy estimate" is perfectly appropriate.